lib/loggerator/rails.rb

```diff
@@ -18,5 +57,8 @@ class Railtie < Rails::Railtie
       end
     end
 
+    config.after_initialize do
+      Loggerator::Rails.setup(::Rails.application)
```

Comment:
As mentioned above, here's an example of where `Loggerator::Rails` is taking precedence over `Rails`, requiring the `::`. This occurs outside of Loggerator as well, causing the same issue in any class that includes Loggerator.